pkg/resources/core/v1/secret/mutator.go

```diff
@@ -126,7 +126,7 @@ func (m *Mutator) admitCreate(secret *corev1.Secret, request *admission.Request)
 // admitDelete checks to see if there are any roleBindings owned by this secret which provide access to a role granting access to this secret
 // if so, it redacts the role so that it only grants delete access. This handles cases where users were given owner access to an individual secret
 // through a controller (like cloud-credentials), and delete the secret but keep the rbac
-func (m *Mutator) admitDelete(secret *corev1.Secret, request *admission.Request) (*admissionv1.AdmissionResponse, error) {
+func (m *Mutator) admitDelete(secret *corev1.Secret, _ *admission.Request) (*admissionv1.AdmissionResponse, error) {
```

Comment:
If this is not implementing an interface, you can just delete the argument altogether.